app/Http/Controllers/ClientController.php

```diff
@@ -12,4 +12,52 @@ public function Client()
         $clients = Client::all();
         return view('client', ['clients' => $clients]);
     }
+
+
+
+public function show($id)
+{
+
+    $data = Client::find($id);
```

Comment:
Recuerda el uso de los repositorios (aplica para los demás controladores).

E.g.

ClientRepository / ClientRepositoryInterface

$this->clientRepository->getClient($id)

Reply:
De acuerdo, realizo el cambio y subo los nuevos cambios

Reply:
Este apartado no me quedo muy claro ethan

Reply:
@YairMedina El uso de repositorios en tus controladores para no tener la lógica directo en tus controladores

- https://www.twilio.com/en-us/blog/repository-pattern-in-laravel-application